src/utils/containercontainer.ts

```diff
@@ -0,0 +1,40 @@
+import { Container } from "../kuberesources.objectmodel";
+import * as explorer from "../explorer";
+import { PodSummary } from "../extension";
+
+export interface ContainerContainer {
+    readonly kindName: string;
+    readonly namespace: string | undefined;
+    readonly containers?: Container[];
+    readonly containersQueryPath: string;
+}
+
+export module ContainerContainer {
```

Comment:
This is a horrible name :P It at least deserves a descriptive comment or a better name (or both :)

Reply:
You would not _believe_ how many people I begged for naming suggestions... and yet somehow they were _all even worse than this_.  I originally had a comment mentioning it but removed it as too self-indulgent!  I'll have another go at coming up with something better, or failing that at least put in a comment to explain what it's for.